thoth/storages/graph/postgres.py

```diff
@@ -4724,6 +4724,48 @@ def get_solver_run_document_ids_all(
 
             return [obj[0] for obj in document_ids]
 
+    def get_python_trove_classifiers_all(
```

Comment:
`get_python_package_version_trove_classifiers_all`?

Reply:
Renamed in a0bd153, PTAL.